metrics/director.go

```diff
@@ -91,4 +91,9 @@ var (
 		Name: "pelican_director_client_version_total",
 		Help: "The total number of requests from client versions.",
 	}, []string{"version", "service"})
+
+	PelicanDirectorGeoIPErrors = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: "pelican_director_geoip_errors",
+		Help: "The total number of errors encountered while downloading the GeoIP database",
```

Comment:
I don't think the help message is quite correct -- it's not really an issue downloading the database (which happens on Director startup, and then twice on Wednesdays and Fridays), but an issue resolving an IP using the database. How about "The total number of errors encountered trying to resolve coordinates using the GeoIP MaxMind database."